analyzer-java/src/test/java/com/infosupport/ldoc/analyzerj/descriptions/AssignmentDescriptionJsonTest.java

```diff
@@ -6,7 +6,7 @@
 import java.io.IOException;
 import org.junit.jupiter.api.Test;
 
-class AssignmentDescriptionJSONTest {
+class AssignmentDescriptionJsonTest {
```

Comment:
I think it might make more sense to keep JSON in caps, as it is an acronym. But if you think this is better that's also fine.

Reply:
I personally don't really have a preference, but the style guide wants it like this:

https://google.github.io/styleguide/javaguide.html#s5.3-camel-case

since it says to 'lowercase *everything* (including acronyms)'.

Reply:
Alright, then it's probably easier to go with the style guide